_includes/projects.html

```diff
@@ -28,15 +28,6 @@
   {% if index_mod == column_index %}
     {% assign starting_point = forloop.index | minus: num_columns %}
     </div>
-
```

Comment:
Removing this section here. It's a redundant link of https://github.com/usds/website/blob/ab885948ac8a16e03a2292abcb99d12f2df75273/_includes/projects.html#L24 that appears to be only for alignment.